README.md

```diff
@@ -65,10 +68,45 @@ You should **NEVER** place arbitrary files in your Windows install folder, no ma
 TODO
 ***
 ###Linux
-TODO (Some distros may have IUP binaries in their package managers, but the author is currently unsure.)
+The Linux binary packages for IUP include both static and dynamic libraries. While efforts are underway to create up-to-date packages for various distributions' package managers, the currently most well supported methods of obtaining IUP binaries are to either compile them from source or download precompiled binaries from the creators.
+
+
+#### Compile from Source
+To compile from source, see [this page][iup-compile]. The instructions to check-out the source tree are available [here][iup-source]. If you understand how to build projects with CMake, then it shouldn't be too difficult.
```

Comment:
@crumblingstatue Yes, but Tecmake is *based on* CMake and there is a Makefile [in the IUP trunk](http://sourceforge.net/p/iup/iup/HEAD/tree/trunk/iup/Makefile), which looks like it can be processed by CMake. Even the build instructions say "just type 'make'".